examples/Cifar10DataTensorFlow.py

```diff
@@ -0,0 +1,39 @@
+import tensorflow as tf
+from aperturedb.TensorFlowData import TensorFlowData
+from typing import List, Tuple
+from PIL import Image
+import io
+
+
+class Cifar10DataTensorFlow(TensorFlowData):
+    """
+    **ApertureDB ingestable Dataset, which is sourced from
+    [Cifar10 (tensorflow.datasets)](https://www.tensorflow.org/datasets/catalog/cifar10)**
+    """
+
+    def __init__(self):
+        (x_train, y_train), (x_val, y_val) = tf.keras.datasets.cifar10.load_data()
+        x = tf.concat([x_train, x_val], axis=0)
+        y = tf.concat([tf.squeeze(y_train), tf.squeeze(y_val)], axis=0)
+        ds = tf.data.Dataset.from_tensor_slices((x, y))
+        super().__init__(ds)
+
+    def __image__from__cifar10__(self, arr):
```

Comment:
Why is this a dunder-method?

Reply:
no need. Infact it should not even be a class method. This has been defined at other places too. I will consolidate and put it in a lib.